src/components/EditFreakForm/__test__/EditFreakForm.test.jsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import EditFreakForm from '../EditFreakForm';
+import { FreakModelDefault, FreakModelKeys } from '../../../models/freaks';
+
+describe('EditFreakForm', () => {
+  const onChange = jest.fn();
+
+  beforeEach(() => {
+    onChange.mockReset();
+  });
+
+  it('should trigger onChange when the firstName is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const firstNameInput = screen.getByTestId('first-name-input');
+    expect(firstNameInput).toBeInTheDocument();
```

Comment:
Asserting that this input is in the document feels like a pretty pointless assertion, the next steps would fail if this wasn't in the document. Delete it.